packages/databricks-vscode/resources/whats-new/1.2.md

```diff
@@ -0,0 +1,48 @@
+# <!-- DATABRICKS LOGO --> Databricks Extension v1.2
+
+## What's new?
+
+-   [📗 Advanced notebook support with Databricks Connect V2](#dbconnect-notebook)
+
+## <a id="dbconnect-notebook"></a> Advanced notebook support with Databricks Connect V2
+
+You could already run notebooks locally using Databricks Connect V2. But we are now introducing advanced notebook support which enables you to run notebooks locally with the similar experience as running them on a Databricks cluster. The aim of this update is to bring most of the convenience of running notebooks on a Databricks cluster to your local environment!
+
+This includes:
+
+-   [Magic commands such as `%run` and `%sql` etc](#magic-commands)
+-   [Preconfigured globals such as spark and dbutils](#preconf-globals)
+-   [Interactive `dbutils.widgets`](#widgets)
+-   [Support for Databricks Notebooks](#dbnb)
+
+### <a id="magic-command"></a>Magic commands
+
+You can now use Databricks magic commands locally from your notebooks.
+
+See [here]() for a list of supported magic commands.
```

Comment:
As a general rule, you should never use `[here]` as link text, here and throughout. This is problematic for accessibility, and it does not help the user confirm that they got to the right target. Instead, try to copy or approximate the title of the target page or article. 

Since I'm not sure what you're specifically trying to link to, I'm not sure how to propose appropriate link text edits here and throughout.

Reply:
Also, you should lead with the "for" statement here and throughout, to help short-circuit readers about whether they need to bother clicking the link. For example: "For a list of supported magic commands, see `[link-text](url)`."